### ISSUE TITLE
Gap between the accuracy of paper reported and that of released model

### ISSUE THREAD

**Issue by jeffreykuang** [clovaai/deep-text-recognition-benchmark#69]
Hi,

    I test the released model TPS-ResNet-BiLSTM-Attn.pth (downloaded from https://drive.google.com/drive/folders/192UfE9agQUMNq6AgU3_E05_FcPZK4hyt), and found that
there exist gaps between the accuracy of paper reported and that of the released model. Details are 
as follows:

![image](https://user-images.githubusercontent.com/17879798/64839881-b4fe1900-d62b-11e9-9478-ba375b7e6512.png).

The test script is as follows:
python -u test.py \
--eval_data  data_lmdb_release/evaluation \
--benchmark_all_eval \
--saved_model  released/TPS-ResNet-BiLSTM-Attn.pth \
--Transformation TPS \
--FeatureExtraction ResNet \
--SequenceModeling BiLSTM \
--Prediction Attn 

Can you explain it?  Thanks.

**Comment by ku21fan** [comment]
Hello, 

The possible reasons are 
1. as we said in the caption,
![스크린샷_091719_042233_PM](https://user-images.githubusercontent.com/7953971/65020071-5b695780-d967-11e9-8b4d-0c37d267b232.jpg)
2. In the paper, we used PyTorch 0.4.1 at that time. 
  In this repository (= pretrained model version), we used PyTorch 1.1.0

Best.
